### PR TITLE
docs: describe comment messages available to test

### PR DESCRIPTION
**Pull request by vchimev**

### CODE REVIEW

File: PULL_REQUEST_TESTS.md [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
-|`tests-ios`            |Executes [tests](https://github.com/NativeScript/nativescript-angular/tree/master/tests) app for iOS.|
-|`testsappng-android`   |Executes [TestsAppNg](https://github.com/NativeScript/tests-app-ng) app for Android.|
-|`testsappng-ios`       |Executes [TestsAppNg](https://github.com/NativeScript/tests-app-ng) app for iOS.|
+### Pull Requests Checks
+
+Builds distributed between Travis CI and internal Jenkins CI execute on pull requests. By default, Travis CI builds trigger on community pull requests and both Travis CI and Jenkins CI builds trigger on core team pull requests. Only NativeScript members have rights to trigger internal Jenkins CI builds. In long term, we aim to migrate all the builds in a public cloud environment.
+
+### Travis
+
+This build compiles the `nativescript-angular` plugin and builds the [tests](https://github.com/NativeScript/nativescript-angular/tree/master/tests) app for Android. Please, refer to the [.travis.yml](https://github.com/NativeScript/nativescript-angular/blob/master/.travis.yml) config file for complete information.
```
Comment (sis0k0) [inline PULL_REQUEST_TESTS.md]: "builds the `nativescript-angular` package" instead

File: PULL_REQUEST_TESTS.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+### Jenkins
+
+More extensive tests execute in Jenkins CI. The table below describes how to trigger and the location of the app/tests of all builds available to execute tests on pull requests.
```
Comment (sis0k0) [inline PULL_REQUEST_TESTS.md]: Add a note again that these can't be triggered externally.

Comment (sis0k0) [reply PULL_REQUEST_TESTS.md]: Mention that to manually trigger a build, you need to comment inside the PR.

**Comment by vchimev** [comment]
router-ios

**Pull request merged**